device/iot-device-samples/multiplexing-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ClientManagerBase.java

```diff
@@ -0,0 +1,166 @@
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.device.DeviceTwin.Pair;
```

Comment:
@timtay-microsoft  , you rightfully mentioned that we shouldn't be using this implementation and there is an out of the box Pair class in java. 
I found javafx.util.Pair implementation ... is that the one you had in mind?

Reply:
My mistake on this one. I thought there was a pair class within Java's collection namespace, but apparently there isn't. This one should be fine, though, sure.

Reply:
For reference, there is an interface defined for a class like Pair in Map: https://docs.oracle.com/javase/8/docs/api/java/util/Map.Entry.html

But the implementation is private, so we have no access to it